ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -122,10 +127,10 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
-    this.initializeProgressMap();
     this.wsNamespace = this.route.snapshot.params['ns'];
     this.wsId = this.route.snapshot.params['wsid'];
     this.creating = this.route.snapshot.queryParams['creating'] || false;
+    this.playground = this.route.snapshot.queryParams['playgroundMode'] || false;
```

Comment:
I commented on my own PR: I'm worried about this actually being a string which might mess things up downstream, please double check with manual testing.